.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Ideally we might want to include our .env files in our gitignore because sometimes this is where we put our confidential keys sometimes also. Is there a reason why you think we should include it in our codebase?

Reply:
yeah we might put this on gitignore, you might wonder what this .env file contains it is a fix on the jest testing since it returning some errors when im trying to run, currently this is the fix they recommended.

https://stackoverflow.com/questions/66891027/how-to-fix-dependency-error-with-react-app-using-jest

Reply:
I see, then in that case then I think it should be okay to keep it as is. We just have to remember why we have the .env file included in our repository